runtime/doc/if_pyth.txt

```diff
@@ -14,16 +14,7 @@ The Python Interface to Vim				*python* *Python*
 6. Tab page objects				|python-tabpage|
 7. vim.bindeval objects				|python-bindeval-objects|
 8. pyeval(), py3eval() Vim functions		|python-pyeval|
-9. Dynamic loading				|python-dynamic|
-10. Python 3					|python3|
-
-{Vi does not have any of these commands}
-
-The Python 2.x interface is available only when Vim was compiled with the
```

Comment:
Sure. I think we should try to unify the nvim-python docs at one point, not totally sure though.